lib/client.js

```diff
@@ -1,5 +1,5 @@
-var PROMISE_TIMEOUT = 5000
-// 5 seconds
+var PROMISE_TIMEOUT = 10000
+// 10 seconds
```

Comment:
Let's reference the ticket here so we have some context if next time we consider changing this again.